.travis.yml

```diff
@@ -2,16 +2,17 @@ language: node_js
 sudo: required
 node_js:
 - '10'
-install:
-- npm ci
-jobs:
-  include:
-  - stage: Lint
-    script: npm run lint
-  - stage: Test
-    script: npm run test && npx codecov
-#  - stage: Deploy
-#    script: npm run deploy && curl -sSL https://raw.githubusercontent.com/RedHatInsights/insights-frontend-builder-common/master/src/bootstrap.sh | bash -s
+cache:
+  yarn: true
+  directories:
+    - node_modules
+install: yarn --frozen-lockfile --non-interactive || (echo 'package.json is not in sync with yarn.lock, check that you include yarn.lock' && false)
+script:
+- yarn && 
+  yarn start && 
```

Comment:
`yarn start` will start the server and wait in loop forever. 
I suggest to change this to `yarn lint` and put each one on a line without the `&&`. So if one step fails, we will have better visibility of where it failed.

Reply:
Thank you. Will do.